### ISSUE TITLE
Extra +1 browser opens in headless: false mode

### ISSUE THREAD

**Issue by d52b8** [thomasdondorf/puppeteer-cluster#301]
Extra +1 browser opens in headless: false mode

For example, if concurrency == 2, it will open 3 instance of browser.

**Comment by Nakavinsky** [comment]
Happening to me aswell, super confusing.

**Comment by mpalavrov** [comment]
I agree here, it is NOT ok for me as well to have 1 blank tab always opened
